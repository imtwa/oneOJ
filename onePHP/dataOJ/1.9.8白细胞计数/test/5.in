122
95.6243 20.329 76.644 49.0673 58.5678 89.1798 36.4636 76.6827 35.0709 92.8299 94.1648 72.6892 22.3076 28.1287 46.7885 76.4064 59.8854 95.9173 23.9197 41.9125 10.7482 44.687 36.5284 74.2715 93.6221 32.5538 96.0665 85.6749 44.4391 54.0109 85.612 27.2346 76.6693 69.3878 67.1755 96.0811 91.7248 89.2055 21.9675 44.2092 38.4723 32.1712 99.4429 80.7432 86.4187 27.9741 17.1041 14.3659 41.1106 23.0304 88.2552 35.2119 52.853 43.4473 31.6774 28.1849 20.9193 97.2747 22.1853 89.2488 92.8226 89.6153 85.3208 49.1154 32.7712 19.1772 19.4137 51.3171 43.1564 99.4228 65.1366 46.0735 56.2295 83.6622 35.1356 29.4136 82.3609 21.1977 88.4191 75.4608 91.4076 42.17 92.5438 50.4949 79.9473 44.1981 72.4672 69.7535 65.8442 67.6608 75.773 57.0371 92.2571 81.7879 16.5797 64.6723 63.9934 30.1926 98.3524 85.2431 91.2014 60.3545 97.5562 42.6029 23.609 99.6146 16.2064 14.9167 73.8537 66.4817 69.7605 95.5595 41.8505 91.8383 52.265 68.3618 64.8102 27.4432 93.8949 58.6347 21.5168 82.6857
