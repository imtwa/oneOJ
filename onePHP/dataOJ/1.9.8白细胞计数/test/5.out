59.64 45.27